requirements.txt

```diff
@@ -1,4 +1,5 @@
 # Library so -e .
 -e .
 boto3
+cllist
```

Comment:
i'm a bit reluctant to introducing new dependencies, can we get by with native python lists/maps for now and fix performance issues when they actually arise? considering how well we're doing on code isolation - migrating executor to different data structures won't be that complicated.